test/desktop/features/fixtures/maze_runner/Assets/Scripts/Main.cs

```diff
@@ -168,6 +175,13 @@ public class Main : MonoBehaviour {
    */
   void RunScenario(string scenario) {
     switch (scenario) {
+      case "OnlyLogBreadcrumbs":
+        Debug.Log("Log");
+        DoUnhandledException(0);
+        break;
+      case "DisableBreadcrumbs":
+        DoUnhandledException(0);
```

Comment:
Should there be a log or something in here to trigger leaving a breadcrumb?

Reply:
there would be a scene loaded breadcrumb